solr/core/src/java/org/apache/solr/cloud/api/collections/BackupCmd.java

```diff
@@ -81,7 +82,7 @@ public void call(ClusterState state, ZkNodeProps message, @SuppressWarnings({"ra
     String backupName = message.getStr(NAME);
     String repo = message.getStr(CoreAdminParams.BACKUP_REPOSITORY);
     boolean incremental = message.getBool(CoreAdminParams.BACKUP_INCREMENTAL, true);
-    String configName = ocmh.zkStateReader.readConfigName(collectionName);
+    String configName = message.getStr(ZkStateReader.COLLECTION_CONFIG_PROP);
```

Comment:
Are you sure that `message` contains the configSet?  (Maybe it does; I don't know).  But what I do know is that you can look it up from DocCollection now.